src/views/requestAccess.js

```diff
@@ -1,146 +1,123 @@
-
-	return (
-		<div className={bottomPad} >
-			<div className={dropDown}>
-			    <div className="dropdown-trigger">
-				    <button className="button is-rounded drop-button" aria-haspopup="true" aria-controls="dropdown-menu" onClick={handleClickDropdown} >
-					    <span className="drop-button-content"> {dropContent} </span>
-					    <span className={dropButton} >
-					    	<i className="fas fa-angle-down" aria-hidden="true"></i>
-					    </span>
-				    </button>
-			    </div>
-			    <div className="dropdown-menu dropdown-menu-place">
-				  	<div className="dropdown-content dropdown-content-width">
-					    <DropDownItem dropContent="JOIN Staff" clickHandle={handleClickJOIN} />
-					    <DropDownItem dropContent="Property Manager" clickHandle={handleClickProperty} />
-				  	</div>
-			    </div>
-			</div>
-			<div>
-				{viewPropertyManagerOptions && <PropertyManagerOptions />}
-				{viewJoinStaffOptions && <JoinStaffOptions />}
-			</div>
-		</div>
-	);
+const RoleDropDown = (props) => {
+  console.log(props.selectionOptions);
+  console.log(typeof props.selectionOptions);
+  let roles = props.selectionOptions;
+  return (
+    <div className="section-row">
+      <div className="select is-rounded">
+        <select
+          defaultValue='default'
+          onChange={e => props.selectionHandler(e.target.value)}>
+          <option value='default' disabled defaultValue>Select Role</option>
+          {
+            props.selectionOptions.map((role) => 
+            <option>{role}</option>
+            )
+          }
+        </select>
+      </div>
+    </div>
+  )
 }
 
 // Section under CONTACT
-export const InfoField = ({label, info}) => {
+export const InfoField = ({label, info, changeHandler}) => {
 	const infoField = "has-text-weight-bold " + ((label === "Phone") ? "phone-field" : (label === "Email" ? "email-field" : "name-field"));
 	return (
 		<div>
 			<hr className="line" ></hr>
 			<span className="input-field"> {label}
-				<p className={infoField} contentEditable>
-			        {info}
-			    </p>
+				<input className={`contact-input ${infoField}`} type="text" defaultValue={info} onChange={(e) => changeHandler(e.target.value)}>
+			    </input>
 		    </span>
 	    </div>
 	);
 }
 
 export const RequestAccess = (props) => {
-	// const id = props.match.params.id;
+  // Get context for API auth header
+  const userContext = useContext(UserContext);
+
+  const [roleObject, setRoleObject] = useState({});
+  const [selectionOptions, setSelectionOptions] = useState([]);
+  const [currentSelection, selectionHandler] = useState("");
+  const [fName, setFirstName] = useState("");
+  const [lName, setLastName] = useState("");
+  const [emailAddress, setEmail] = useState("");
+
+  useEffect(() => {
+    console.log(process.env.REACT_APP_API_URL)
+    axios.get(`/api/roles`)
```

Comment:
I'm getting a 404 when testing this since /roles does not exist in the backend 
![image](https://user-images.githubusercontent.com/8600792/90591926-22e68500-e199-11ea-8180-4cb20907d65b.png)

Let's comment this line out and put in some dummy data (which won't really be that different from the real data) until we get that API endpoint setup